### ISSUE TITLE
Publish images for sample applications used in the App Mesh examples walkthroughs

### ISSUE THREAD

**Issue by fawadkhaliq** [aws/aws-app-mesh-examples#330]
**Description**
Currently, every time a user wants to try any of the walkthrough examples in this repo, they need to build the sample application images, publish them to ECR and then use them in their App Mesh setup. This also requires the users to have Docker installed on their machine to build the images.

**Proposed change**
We should publish images for sample applications used in the App Mesh walkthroughs and by default use the published images. If a user wants to modify the application code or override, they should have the ability to do so.

**Comment by jlbutler** [comment]
We should consider smaller images as well. It should be fairly easy to drop some bulk on these. I can help with this, later this week @fawadkhaliq.

**Comment by fawadkhaliq** [comment]
@jlbutler absolutely! That'd be great if you can help with it. thanks!

**Comment by fawadkhaliq** [comment]
@jlbutler assigning the issue to you since you'll be working on it

**Comment by jlbutler** [comment]
Yep I'm working on this one... in slow motion, apparently. I have been wondering if we should also port the demo app that most of these use to Go for a faster build and smaller image. I will update again soon and get something rolling.

**Comment by jlbutler** [comment]
Status update, I'm working through an internal process on releasing these images as artifacts. Once that's complete, I can move on to resolving this.

**Comment by jlbutler** [comment]
Another update so the issue doesn't go stale. The process is well defined, but it'll take some refactoring and some legwork in the repo across all of the walkthroughs. I have scheduled time to work on this once I'm through a set of current work and will bring back a proposal PR here.
